rails-app-with-knapsack_pro/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 8.1.1"
+# NOTE: Need edge Rails to run minitest v6 (https://github.com/rails/rails/pull/56202)
+gem "rails", github: "rails/rails", branch: "main"
```

Comment:
question

If you use Rails (not from the main branch) + Minitest 5.x, are CI tests still passing? 

I assume the PR is backward compatible.

Reply:
Yes, they pass: 

https://github.com/KnapsackPro/knapsack_pro-ruby/pull/321/commits/f528d09f53b54b9a9077b8e8b5b504c66d0b32b9